modules/iopcore/cdvdman/ncmd.c

```diff
@@ -66,6 +66,30 @@ void lba_to_msf(s32 lba, u8 *m, u8 *s, u8 *f)
     *f = lba % 75;
 }
 
+//-------------------------------------------------------------------------
+typedef struct
+{
+    u8 ctrl_adr;
+    u8 track_no;
+    u8 point;
+    u8 min;
+    u8 sec;
+    u8 frm;
+    u8 zero; // always zero
+    u8 abs_min;
+    u8 abs_sec;
+    u8 abs_frm;
+} __attribute__((packed)) toc_point_t;
+
+typedef struct
+{
+    toc_point_t a0;
+    toc_point_t a1;
+    toc_point_t a2;
+    toc_point_t track[99]; // commented for saving space, uncomment when CDDA support added
```

Comment:
I think this comment is useless. We never allocate a buffer for the TOC, we only point to a buffer that has already been allocated by the game and it must always be the same structure.

Reply:
yes, it was left from first design, just forgot to remove it